dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/UninitializedEventDescriptor.scala

```diff
@@ -0,0 +1,65 @@
+package org.bitcoins.dlc.oracle
+
+import org.bitcoins.core.number._
+import org.bitcoins.core.protocol.tlv._
+import org.bitcoins.crypto.SchnorrNonce
+
+sealed trait UninitializedEventDescriptor[+T <: EventDescriptorTLV] {
```

Comment:
Can you add a scaladoc on this? I don't really understand what this means. What does it mean if an event descriptor is uninitialized?

Reply:
Is this basically an `UnitializedEventBuilder` similar to a `VectorBuilder` ?

Reply:
Since event descriptors have the nonces in them we can't build a descriptor without first initializing it in the oracle. The idea for this type is to hold all the data for an event descriptor minus the nonces.

Reply:
Ok so my comment is right? It's really a `EventDescriptorBuilder`?

Reply:
also scaladoc please!!

Reply:
> Ok so my comment is right? It's really a EventDescriptorBuilder?

Not really, you have to add all the nonces at once. Maybe I should rename it to `setNonces`